etcdctl/command/member_commands.go

```diff
@@ -86,7 +86,10 @@ func actionMemberAdd(c *cli.Context) {
 	mAPI := mustNewMembersAPI(c)
 
 	url := args[1]
-	ctx, cancel := context.WithTimeout(context.Background(), client.DefaultRequestTimeout)
+	totalCtx, totalCancel := contextWithTotalTimeout(c)
+	defer totalCancel()
+
+	ctx, cancel := contextWithPerRequestTimeout(c, totalCtx)
```

Comment:
Yes, I misused `client.DefaultRequestTimeout` :( Thanks for pointing. I'll fix it.